src/controller/python/chip/ChipDeviceCtrl.py

```diff
@@ -423,6 +455,7 @@ def CommissionWithCode(self, setupPayload: str, nodeid: int):
         return self._ChipStack.commissioningEventRes == 0
 
     def CommissionIP(self, ipaddr: str, setupPinCode: int, nodeid: int):
+        """ DEPRECATED, DO NOT USE! Use `CommissionOnNetwork` or `CommissionWithCode` """
```

Comment:
Can't seem to find `CommissionOnNetwork`?

Reply:
See line 400. It was there before.